msteams-platform/bots/how-to/conversations/bot-copilot-handoff.md

```diff
@@ -0,0 +1,178 @@
+
+:::image type="content" source="../../../assets/images/Copilot/Copilot-handoff.gif" alt-text="The GIF shows the conversation handoff between the Copilot for Microsoft 365 and the Aisera chat bot.":::
+
+## How Copilot handoff works
+
+A custom Copilot sends a deep link with a continuation token to the Copilot for Microsoft 365. The deep link query parameter with a continuation token ensures that any information from your plugin invocation parameters is referenced.  When the user selects the deep link, the Copilot then sends an invoke call to the bot with the continuation token, and the bot can resume the conversation based on the context. This process enables a seamless transition from Copilot to a custom Copilot, maintaining the conversation’s continuity and context, which optimizes the user experience.
+
+:::image type="content" source="../../../assets/images/Copilot/copilot-handoff-flow.png" alt-text="Screenshot shows the handoff flow between the user, Copilot, plugin, Teams, and bot." lightbox="../../../assets/images/Copilot/copilot-handoff-flow.png":::
+
+You must create a deep link URL with a `continuation` query parameter for the action button and assign a continuation token to the parameter to facilitate the handoff process. When the user selects the action button, Microsoft Teams reads the continuation token from the URL and initiates an invoke call to the bot. The bot or plugin uses the continuation token to create a response, which is then displayed to the user in the plugin chat window.
+
+## Enable copilot handoff
+
+To enable Copilot handoff in Teams, follow these steps:
+
+1. **Configure a deep link URL**: Create a deep link to the bot chat and include `botMri` and `continuationToken`. The deep link must be in the format `https://teams.microsoft.com/l/chat/0/0?users=${botMri}&continuation=${continuationToken}`.
+
+   **Example**:
+
+   ```JSON
+   { 
+
+   "type": "Action.OpenUrl", 
+
+   "title": "Handoff to Bot", 
+
+   "url": "https://teams.microsoft.com/l/chat/0/0?users=${botMri}&continuation=${continuationToken}" 
+   }
+   ```
+
+   **Parameters**
+
+   |Property |Description  |
+   |---------|---------|
+   |`botMri`     |The `botMri` must be in the format `28:<botID>`. The botMri specifies the bot chat that the user is redirected to when they select the action button that contains the deep link url.        |
+   |`continuationToken`     | The `continuationToken` value can be any string, but ensure that the total URL length doesn't exceed 2048 characters and the string is URL encoded.        |
+   |`Action.OpenUrl`     |  The `Action.OpenUrl` property allows the user to handoff the conversation to a bot.       |
+
+   **Sample payload**
```

Comment:
What is this a sample payload of? It seems to be of the `handoff/action` invoke, but you haven't introduced that invoke type yet.

Reply:
Update as "The bot receives the continuation token from the URL in the form of an invoke and uses the token to maintain the context of the conversation."